Sources/AccountSDKIOSWeb/Lib/API/SchibstedAccountAPI.swift

```diff
@@ -69,13 +69,17 @@ class SchibstedAccountAPI {
         return requestWithHeaders
     }
     
-    func sessionExchange(for user: User, clientId: String, redirectURI: String, completion: @escaping HTTPResultHandler<SessionExchangeResponse>) {
+    func sessionExchange(for user: User, state: String? = nil, clientId: String, redirectURI: String, completion: @escaping HTTPResultHandler<SessionExchangeResponse>) {
```

Comment:
lets have the same order of params. altug requested after redirectURI, since they are connected, so will do that on android :)

Reply:
Good point! the order should be also changed in webSessionURL method